config/secrets.yml

```diff
@@ -19,9 +19,13 @@
 
 development:
   secret_key_base: d22cd95910c4425ab8adfd2aa968b845775c6ffff23f9cdb337bc6f210199ca97736ed261d06bf69242f6483a8e6c474ab88f02dabb9865f9911ff215da31695
-
+  slack_token: "xoxp-235986621104-238493557106-238649524195-b886dacfa83620d7c79b483114198cf9"
+  slack_channel: "#shelters_notif"
```

Comment:
This could be good as a environment variable too, even if it isn't a secret per se. That'd make it easy to test in development.